packages/parameterized_test/README.md

```diff
@@ -11,79 +11,172 @@ and the Flutter guide for
 [developing packages and plugins](https://flutter.dev/developing-packages). 
 -->
 
-# Parameterized test
+# 🧪 Parameterized test
```

Comment:
It feels a bit weird to have multiple README's with the same contents. Isn't there a way to redirect to the top level readme cleanly?

Reply:
Maybe you can copy the Readme.md with github actions or commit hooks?

Reply:
In the past I did some symlinking but that wasn't perfect either. I'm planning on removing the packages structure. Since the other packages won't be need afterwards when they are discontinued. When thats done it will be 1 readme